packages/render-fidelity-tools/src/common.ts

```diff
@@ -139,26 +140,17 @@ export class ImageComparator {
     image[position + 3] = a;
   }
 
-  analyze(threshold: number, options: AnalysisOptions = {
```

Comment:
analyze and generate visual are in the same functions before, probably because they both loop through all pixels in a given threshold. Now i remove threshold in analyze function, if feel like they have less in common and just seperate them.